nix/lanzaboote.nix

```diff
@@ -32,9 +32,11 @@ in
   };
 
   config = mkIf cfg.enable {
-    # bootspec is putting at false
-    # until we fix this upstream, we will mkForce it.
-    boot.loader.supportsInitrdSecrets = mkForce true;
+    boot.bootspec = {
+      enable = true;
+      extensions."org.lanzaboote.osRelease" = config.environment.etc."os-release".source;
```

Comment:
Why do we have `org.` here? Seems superfluous

Reply:
It seems to me to be leaning into the relative "standard" of reverse domain names as package identifiers and whatnot.

Reply:
Yeah, this is very "corporate" namespacing, it could have been `com.lanzaboote.osRelease` :>
I have no strong feeling on that as we did not specify what to expect

Other than that, I like to think of `org` as something which would be for public extensions, and private extensions could have another prefix or whatever, but really, I have no strong feeling

Reply:
I agree that org is better than com. But it still feels very javaesque. I'd vote to just get rid of any further qualifier and leave it at `lanzaboote.osRelease`